src/index.ts

```diff
@@ -48,6 +48,7 @@ export {
 } from './theme';
 export { Card } from './Card';
 export { Hero } from './Hero';
+export { Tabs } from './Tabs';
```

Comment:
Do we need entries for `Tab` and `Pane`?

Reply:
Also, we may need entries in the rollup config.

Reply:
Hmm. `Pane` is rolled up like `Hero` e.g.
```
const { Pane } = Tabs;
```
so it doesn't need to be exported here or in rollup to be consumed. `Tab` on the other hand I'm not sure if I want to export. It isn't needed for the consumer because its auto generated from the panes. It perhaps doesn't need to be in it's own folder/file.